client/state/plans/selectors.js

```diff
@@ -17,3 +22,45 @@ export const getPlans = state => {
 export const isRequestingPlans = state => {
 	return state.plans.requesting;
 };
+
+/**
+ * Returns a plan
+ * @param  {Object} state      global state
+ * @param  {Number} productId  the plan productId
+ * @return {Object} the matching plan
+ */
+export const getPlan = createSelector(
+	( state, productId ) => getPlans( state ).filter( plan => plan.product_id === productId ).shift(),
```

Comment:
Nice. Why is `.shift()` necessary here? Can there be two plans with the same product_id?